.prettierignore

```diff
@@ -5,3 +5,5 @@ cfgov/agreements/static/agreements/js/jquery-3.5.1.min.js
 cfgov/static_built/
 cfgov/legacy/static/nemo/_/js/
 collectstatic/
+*.less
+*.css
```

Comment:
Wait, I don't think we want to ignore these?

Reply:
Running prettier instead via the prettier stylelint plugin

Reply:
FWIW Prettier [docs](https://prettier.io/docs/en/integrating-with-linters.html#notes) appear to discourage this:

> These plugins [stylelint-prettier] were especially useful when Prettier was new. By running Prettier inside your linters, you didn’t have to set up any new infrastructure and you could re-use your editor integrations for the linters. But these days you can run `prettier --check `. and most editors have Prettier support.

And the plugin [recommendations](https://github.com/prettier/stylelint-prettier#recommended-configuration) say to basically wipe the whole `.stylelintrc` and only have `{ "extends": ["stylelint-prettier/recommended"] }`, or use `prettier-stylelint` instead.